src/modal-manager/ModalManager.module.scss

```diff
@@ -0,0 +1,106 @@
+:local {
+  .container {
+    height: 100%;
+  }
+
+  .modal-manager {
+    background-color: var(--terra-application-modal-manager-background-color, #fff);
```

Comment:
@tbiethman @ryanthemanuel  Is there someone working on filling in the new theme variables?

Reply:
This has been removed.